docs/003-markers.md

```diff
@@ -0,0 +1,441 @@
+        <td>The marker sequence identifier of the marker sequence.</td>
+    </tr>
+    <tr>
+        <td>parentReferences</td>
+        <td>map[uint64]Thresholdmap</td>
+        <td>The marker referenced map of each parent marker.</td>
+    </tr>
+    <tr>
+        <td>rank</td>
+        <td>uint64</td>
+        <td>The rank of the marker sequence in the marker sequence DAG.</td>
+    </tr>
+    <tr>
+        <td>highestIndex</td>
+        <td>uint64</td>
+        <td>The highest MI of the marker sequence.</td>
+    </tr>
+    <tr>
+        <td>lowestIndex</td>
+        <td>uint64</td>
+        <td>The lowest MI of the marker sequence.</td>
+    </tr>
+</table>
+
+
+#### Create Marker Sequence
+A new marker sequence is created when:
+1. there's a conflict in a UTXO branch.
+2. the UTXO branches are aggregated.
+3. UTXO branches are merged.
+
+Each new marker sequence starts from a new marker.
+
+### The Markers
+Markers are messages selected from the tip set periodically and assigned unique identifiers, in the form of *`[MSI, MI]`*. 
+
+#### Marker Structure
+<table>
+    <tr>
+        <th>Name</th>
+        <th>Type</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+        <td>MarkerSequenceID</td>
+        <td>uint64</td>
+        <td>The marker sequence identifier of the marker.</td>
+    </tr>
+    <tr>
+        <td>MarkerIndex</td>
+        <td>uint64</td>
+        <td>The index of the marker in the marker sequence.</td>
+    </tr>
+</table>
+
+#### Create Markers
+A new marker is created when:
+1. the default conditions are met:
+    * **every x messsages**;
+    + **every t seconds**;
+    + a mix of the first two!
+        + Upperbound given by the messages
+        + Lower temporal bound given by the time
+    + every x messages that reference (directly or indirectly) the previous marker
+        + Lower bound given by rank (e.g., how far you are in terms of steps) -> >= 10 or something
+        + Upper bound given by the amount of messages referencing the previous one -> ~ 200 msgs
+2. A new marker sequence is created. 
+> :mega: to be confirmed here.
+ 
+A new marker is selected from the tips set randomly, and selected from the weak tips set if there's no strong tip. A new pair of *`[MSI, MI]`* is assigned to the new marker. 
+> :mega:  to be confirmed here.
+
+
+The *`MSI`* is set according to the following rules:
+* Create a new MSI if it is the first marker of the new marker sequence.
+* Inherit the MSI from parents if it references the latest marker of the sequence and meets the requirement to set up a new marker.
+
+The *`MI`* is set to *`MI = 1+ max(referencedMI)`*, which complies to the rule:
++ Marker indexes (*`MIs`*) are monotonically increasing such that *`\forall x \in fc(y)`* => *`MI_x > MI_y`*, where *`fc(y)`* is the future cone of *`y`* and *`x`* is any message in that future cone.
+
+### Past Markers and Future Markers
+Each message keeps the marker information in two lists:
+* past markers (PMs)
```

Comment:
PM and FM are already in use for past marker and future marker. (Typically if you want to talk about past markers you would say PMs - hence the name PMs is already reserved). This will create confusion. Below in the table you call the lists FutureMarkers and PastMarkers. this would be better.